flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinatorTest.java

```diff
@@ -373,4 +405,35 @@ public void processWatermark(Watermark mark) throws Exception {
 		public void processLatencyMarker(LatencyMarker latencyMarker) {
 		}
 	}
+
+	private static SubtaskCheckpointCoordinator coordinator(boolean unalignedCheckpointEnabled, ChannelStateWriter channelStateWriter) throws IOException {
+		return new SubtaskCheckpointCoordinatorImpl(
+			checkpointStorageWorkerView(100),
```

Comment:
nit: can use `new TestCheckpointStorageWorkerView(100)` for simple.

Reply:
Good point!